packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -162,47 +169,79 @@ export class MirrorNodeClient {
             registers: [register]
         });
 
-        this.logger.info(`Mirror Node client successfully configured to ${this.baseUrl}`);
+        this.logger.info(`Mirror Node client successfully configured to REST url: ${this.restUrl} and Web3 url: ${this.web3Url} `);
     }
+    
+    private buildUrl(baseUrl: string) {
+        if (!baseUrl.match(/^https?:\/\//)) {
+            baseUrl = `https://${baseUrl}`;
+        }
+
+        if (!baseUrl.match(/\/$/)) {
+            baseUrl = `${baseUrl}/`;
+        }
 
-    async request(path: string, pathLabel: string, allowedErrorStatuses?: number[], requestId?: string): Promise<any> {
+        return `${baseUrl}api/v1/`;
+    }
+
+    private async request(path: string, pathLabel: string, method: REQUEST_METHODS, data?: any, allowedErrorStatuses?: number[], requestId?: string): Promise<any> {
         const start = Date.now();
         const requestIdPrefix = formatRequestIdMessage(requestId);
         let ms;
         try {
-            const response = await this.client.get(path, {
-                headers:{
-                    'requestId': requestId || ''
-                }
-            });
-            ms = Date.now() - start;
-            this.logger.debug(`${requestIdPrefix} [GET] ${path} ${response.status} ${ms} ms`);
+            let response;
```

Comment:
nit: Define headers once
```suggestion
            let response;
            const headers = {
                    headers:{
                        'requestId': requestId || ''
                    }
                };
```